docs/local-gadget.md

```diff
@@ -2,14 +2,63 @@
 title: local-gadget
 weight: 80
 description: >
-  How to use the local-gadget tool.
+  Description of the local-gadget tool.
 ---
 
-Inspektor Gadget can also be used without Kubernetes to trace containers with
-the `local-gadget` tool.
+Inspektor Gadget relies on the Kubernetes API server to work. However, there are
+[some cases](#use-cases) where it is necessary, or preferred, to trace
+containers without passing through Kubernetes. In such scenarios, you can use
+the `local-gadget` tool, as it allows you to collect insights from the nodes to
+debug your Kubernetes containers without relying on Kubernetes itself, but on
+the container runtimes. It is important to remark that `local-gadget` can also
+be used to trace containers that were not created via Kubernetes.
+
+Some characteristics of the `local-gadget`:
+- Uses eBPF as its underlying core technology.
+- Enriches the collected data with the Kubernetes metadata.
+- Easy to install as it is a single binary (statically linked).
+
+## Use cases
+
+- In a Kubernetes environment, when the Kubernetes API server is not working
+  properly, we cannot deploy Inspektor Gadget. Therefore, we still need a way to
+  debug the containers of the cluster.
+- If you are developing an application that needs to get insights from the
+  Kubernetes node, you have two options:
+  - Include the `local-gadget` binary in your container image, and your
+    app simply execs `local-gadget` (using JSON format to ease the parsing).
+  - If your application is in Go, you can use the internal packages of
+    `local-gadget`, see [architecture]() for further details.
+- Outside a Kubernetes environment, for observing and debugging standalone
+  containers.
+
+
+## Architecture
```

Comment:
We already have an architecture document (https://github.com/kinvolk/inspektor-gadget/blob/main/docs/architecture.md) (it's very outdated unfortunately). What about adding this there?